pallets/admin-utils/src/lib.rs

```diff
@@ -1238,6 +1248,34 @@ pub mod pallet {
             ChainId::<T>::set(chain_id);
             Ok(())
         }
+
+        /// A public interface for `pallet_grandpa::Pallet::schedule_grandpa_change`.
+        ///
+        /// Schedule a change in the authorities.
+        ///
+        /// The change will be applied at the end of execution of the block `in_blocks` after the
+        /// current block. This value may be 0, in which case the change is applied at the end of
+        /// the current block.
+        ///
+        /// If the `forced` parameter is defined, this indicates that the current set has been
+        /// synchronously determined to be offline and that after `in_blocks` the given change
+        /// should be applied. The given block number indicates the median last finalized block
+        /// number and it should be used as the canon block when starting the new grandpa voter.
+        ///
+        /// No change should be signaled while any change is pending. Returns an error if a change
+        /// is already pending.
+        #[pallet::call_index(59)]
+        #[pallet::weight(<T as Config>::WeightInfo::swap_authorities(next_authorities.len() as u32))]
+        pub fn schedule_grandpa_change(
+            origin: OriginFor<T>,
+            // grandpa ID is always the same type, so we don't need to parametrize it via `Config`
+            next_authorities: AuthorityList,
+            in_blocks: BlockNumberFor<T>,
```

Comment:
The parameters come from this function: https://paritytech.github.io/polkadot-sdk/master/pallet_grandpa/pallet/struct.Pallet.html#method.schedule_change 

`in_blocks` is since how many blocks from the current one the change will be applied.

It definitely better to have it like an function/extrinsic parameter rather than a `Config`'s const. For the const we would need to upgrade the runtime to adjust this parameter.